test_api.py

```diff
@@ -1196,7 +1196,7 @@ def test_culled_host_is_removed(self, emit_event):
         self._run_host_reaper()
         self._check_hosts_are_deleted((added_host_id,))
 
-        events = tuple(json.loads(event) for event in emit_event.events)
+        events = tuple(json.loads(event[0]) for event in emit_event.events)
```

Comment:
We can get rid of the magic number by unpacking the tuple.

```suggestion
        events = tuple(json.loads(event) for event, key in emit_event.events)
```